Minor improvements

Spappz opened the pull request:
- Fixed a tiny typo in the renderer demo
- DA afflictions with Tempted Curse text now display their rules in the Quick Rules tab
- **Revamped `compact` `ability` entries.** The existing code was quite buggy, placing spurious semicolons all over the place and not really matching the source. I've [trawled as much book as possible](https://docs.google.com/spreadsheets/d/1iaf2vaeysj74O1vCykj_qp2-_1QryiezpMpeE31eAzg/edit) to work out the Definitive™ style guide (plus some extrapolations where I could find no exact correspondence), so you can actually verify the work!

I was going to start sifting through the hazards page for regularity and correctness but I'll save that for another time. 😏

MrVauxs commented:
Feel free to leave a changelog note describing the changes and I'll merge it afterward.

Spappz commented:
smh i was just gonna update the changelog file directly 😩

- Finished adding The Slithering
- Added filter and quick rules for Tempted Curse afflictions
- Improved rendering of compact abilities
- Fixed rendering bugs for creature spellcasting and known rituals text
- (Typos/Tags)

---

I went and fixed a bunch more typos (including the outstanding ones from #typos channel, save the one about thaumaturge's skills) and did the other things above to make it a little meatier. 😎

#131 can be closed now; 95% of it was already there but I've added the remainder and fixed a whole bunch of typos in it. Also, this experience has made me also greatly fearful of the creature schema, so perhaps I might suggest some prodding at that later. At the very least, quite a bit of the rendering script seems needlessly complicated and irregular, but I didn't do anything more than one-line typo fixes here. Too much to think about for just a tack-on!

MrVauxs commented:
Yeah we are in a swath of tech debt that we will have to eventually dig ourselves out from. The Text Converter has been a push towards that but then we just stagnated with that momentum.
